homeassistant/components/apsystemsapi_local/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "apsystemsapi_local",
```

Comment:
Please change the domain to `apsystems`

Reply:
There would also be the option the query the data using their cloud. What do you think about apsystems_local?

Reply:
if we were to add a cloud one, we can call that one `apsystems_cloud`. We don't make concessions here for clouds ;-)

Reply:
I think that we should call it `apsystems` and if a cloud one is added, we can call that one `apsystems_cloud`